kubernetes-filebeat/templates/configmap.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kubernetes-filebeat.fullname" . }}-config
+  labels:
+    {{- if .Values.cluster_service }}
+    k8s-app: {{ template "kubernetes-filebeat.fullname" . }}
```

Comment:
Why do you introduce a new label ?

Reply:
k8s-app is a special one; it allows labeled apps to appear in the "cluster-info"